typings/index.d.ts

```diff
@@ -524,6 +530,7 @@ declare module 'discord.js' {
     MessageTypes: MessageType[];
     SystemMessageTypes: SystemMessageType[];
     ActivityTypes: ActivityType[];
+    StickerFormatTypes: StickerFormatTypes;
     OverwriteTypes: OverwriteTypes;
```

Comment:
```suggestion
    StickerFormatTypes: typeof StickerFormatTypes;
    OverwriteTypes: typeof OverwriteTypes;
```

Reply:
Is something wrong with that? I think it shouldn't have `typeof`

Reply:
Without it, the property would have the type of one of the enum's values, but because this is the enum itself, you need `typeof` there

Reply:
Oh it was an enum. Is it okay to edit code non-related to this PR?

Reply:
Normally you shouldn't but it's fine in this case.